src/components/WizardPattern/WizardPattern.js

```diff
@@ -0,0 +1,215 @@
+
+    const prevStepUnreachable = onFirstStep || this.getPrevStep(); // || ??
+    const nextStepUnreachable = nextStepDisabled; // || ??
+
+    return (
+      <Modal
+        show={show}
+        onHide={onHide}
+        onExited={onExited}
+        dialogClassName="modal-lg wizard-pf"
+      >
+        <Wizard>
+          <Modal.Header>
+            <button
+              className="close"
+              onClick={onHide}
+              aria-hidden="true"
+              aria-label="Close"
+            >
+              <Icon type="pf" name="close" />
+            </button>
+            <Modal.Title>{title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body className="wizard-pf-body clearfix">
+            <WizardPatternBody
+              {...this.props}
+              goToStep={this.goToStep}
+              nextStepDisabled={nextStepDisabled}
+              stepButtonsDisabled={stepButtonsDisabled}
+              activeStepStr={activeStepStr}
+            />
+          </Modal.Body>
+          <Modal.Footer className="wizard-pf-footer">
+            <Button bsStyle="default" className="btn-cancel" onClick={onHide}>
+              {cancelButtonText}
+            </Button>
+            <Button
+              bsStyle="default"
+              onClick={this.onBackClick}
+              disabled={prevStepUnreachable}
+            >
+              <Icon type="fa" name="angle-left" />
+              {backButtonText}
+            </Button>
+            <Button
+              bsStyle="primary"
+              onClick={onFinalStep ? onHide : this.onNextClick}
+              disabled={nextStepUnreachable}
+            >
+              {onFinalStep ? (
+                closeButtonText
+              ) : (
+                  <React.Fragment>
+                    {nextButtonText}
+                    <Icon type="fa" name="angle-right" />
+                  </React.Fragment>
+                )}
+            </Button>
+          </Modal.Footer>
+        </Wizard>
+      </Modal>
+    );
+  }
+}
+
+WizardPattern.propTypes = {
+  activeStepIndex: PropTypes.number.isRequired,
+  show: PropTypes.bool,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  onHide: PropTypes.func,
+  onExited: PropTypes.func,
+  onBack: PropTypes.func,
+  onNext: PropTypes.func,
+  onStepChanged: PropTypes.func,
+  loadingTitle: PropTypes.string,
+  loadingMessage: PropTypes.string,
+  loaded: PropTypes.bool,
+  cancelButtonText: PropTypes.string,
+  backButtonText: PropTypes.string,
+  nextButtonText: PropTypes.string,
+  closeButtonText: PropTypes.string,
+  steps: PropTypes.arrayOf(PropTypes.shape(stepShape)),
+  nextStepDisabled: PropTypes.bool,
+  stepButtonsDisabled: PropTypes.bool
+};
+
+WizardPattern.defaultProps = {
+  show: false,
+  title: '',
+  onHide: noop,
+  onExited: noop,
+  onBack: noop,
+  onNext: noop,
+  onStepChanged: noop,
+  loadingTitle: 'Loading Wizard...',
+  loadingMessage: 'Loading...',
+  loaded: false,
```

Comment:
consider the use of [`loading` here, matches with the spinner component](https://github.com/patternfly/patternfly-react/blob/b6f5027f38a5ee9afeaf8be1da0a875939234594/src/components/Spinner/Spinner.js#L36)

Reply:
Okay, cool, good to know